cosmoz-grouped-list.js

```diff
@@ -177,12 +180,35 @@
 				templateInstance = this._getModelFromItem(item);
 				if (templateInstance) {
 					itemPath = ['item'].concat(pathArray.slice(2));
-					templateInstance.notifyPath(itemPath, value);
+					itemPath = IS_V2 ? itemPath.join('.') : itemPath;
```

Comment:
I think using path arrays was a mistake here. Especially when looking at `notifyPath` code: according to the API, it expects a string (it accepts path array though but perform extra work on it).
So my recommendation here would be to adhere to the API, and use only path strings.

Reply:
@plequang I will rewrite this to use strings.